plugins/grafana-sankey-plugin/src/SankeyPanel.tsx

```diff
@@ -53,11 +53,11 @@ export const SankeyPanel: React.FC<Props> = ({ options, data, width, height }) =
           destination = destinationIPs[i];
         }
       } else {
-        // Google Chart will not be rendered if source is equal to destination.
-        // Add an extra space to differentiate to-self traffic (e.g. intra-Node).
-        if (source === destination) {
-          destination = destination + ' ';
-        }
+        // Google Chart will not render if cycle exists. Explicit cycle: a->a
+        // (e.g. intra-Node traffic in node_to_node_flows dashboard); Implicit
+        // cycle: a->b, b->c, c->a. Add an extra space to all the destination
+        // names to avoid introducing cycles.
+        destination = destination + ' ';
```

Comment:
I think rather than "implicit" and "explicit", these are called "indirect" and "direct" cycles?